docs/FAQ.md

```diff
@@ -1,53 +1,57 @@
-are two primary schools of thought: Static Root of Trust Measurements and
-Dynamic Root of Trust Measurements.
+Like our [Glossary](./theory/Glossary.md#trust) says,
+trust is an assured reliance on the properties, ability, strength, or truth of
+an entity. If we assume any information or property is true without measuring
+and proving it, we need to trust it. This might include the belief, that our
+computer will behave in an expected way, that it won't leak our data, that it
+won't do something we didn't intend to do.
+
+When establishing the trust of system components there are two primary schools
+of thought: Static Root of Trust for Measurement (SRTM) and Dynamic Root of Trust
+for Measurement (DRTM).
 
 ### SRTM
 
-SRTM takes place at system boot from firmware / bios. This process requires
-measurements of the entire boot sequence at once. This is both a good and a
-bad thing, because any changes to any part of the process (including even
-bios settings) would cause the entire system to need to be remeasured. SRTM
-Starts with Core Root of Trust for Measurement (CRTM), and works its way up the chain where
-each link verifies the next. In other words, SRTM starts by measuring the bios
-boot block and then moves on to measure the rest of the bios.
+SRTM Starts with Core Root of Trust for Measurement (CRTM), and works its way
+up the chain where each link verifies the next. In other words, SRTM starts by
+measuring the BIOS boot block and then moves on to measure the rest of the BIOS.
+SRTM has a few problems:
+
+- Large TCB (Trusted Computing Base) - the more software as a part of the chain
+of trust, the more potential vectors of attack.
+- Takes place before boot - reassuring the validity of the chain of trust
+requires a reboot.
+- Only provides load-time integrity assurance - the integrity was maintained
+during boot, but there is no guarantee it will stay like that.
 
 ### DRTM
 
 DRTM aims to address shortcomings of SRTM. DRTM creates trusted environment from
-an untrusted state. Essentially what this means is that regardless of weather
+an untrusted state. Essentially what this means is that regardless of whether
 or not computer was booted from trusted software, it uses hardware to ensure
-that no malicious software has been run. For this reason, DRTM takes place
-after boot, as opposed to SRTM, which takes place pre-boot. Intel's solution
-for DRTM is TXT, which utilizes special instructions on the processor (SMX).
-AMD's solution for DRTM is called SKINIT. Trenchboot provides a solution for
-setting up DRTM for an arbitrary platform.
+that no malicious software, which may or may not have been run before the
+dynamic launch, can impact software started after the dynamic launch. For this
+reason, DRTM takes place after boot, as opposed to SRTM, which takes place
+pre-boot. Intel's solution for DRTM is TXT, which utilizes special instructions
+on the processor (SMX). AMD's solution for DRTM is called SKINIT. TrenchBoot
```

Comment:
"Overall, TrenchBoot provides a solution for setting up DRTM on an arbitrary platform."